src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -82,4 +94,58 @@ public String getSignerName() {
     public String getSignerRole() {
         return signerRole;
     }
+
+    private void setAmountToPayByDefendant(
+        final InterestCalculationService interestCalculationService,
+        final Claim claim
+    ) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        if (countyCourtJudgment.getPaymentOption() == PaymentOption.INSTALMENTS) {
+            countyCourtJudgment.getRepaymentPlan().ifPresent(
+                repaymentPlan -> this.amountToPayByDefendant = formatMoney(repaymentPlan.getRemainingAmount())
+            );
+        } else if (countyCourtJudgment.getPaymentOption() == PaymentOption.FULL_BY_SPECIFIED_DATE) {
+            this.amountToPayByDefendant = calculateClaimAmountWithInterest(
+                interestCalculationService,
+                claim,
+                countyCourtJudgment.getPayBySetDate().orElse(null)
+            );
+        } else if (countyCourtJudgment.getPaymentOption() == PaymentOption.IMMEDIATELY) {
+            this.amountToPayByDefendant = calculateClaimAmountWithInterest(
+                interestCalculationService,
+                claim,
+                claim.getCountyCourtJudgmentRequestedAt().toLocalDate()
+            );
+        }
+    }
+
+    private String calculateClaimAmountWithInterest(
+        final InterestCalculationService interestCalculationService,
+        final Claim claim,
+        final LocalDate toDate
+    ) {
+        BigDecimal claimAmount = ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount();
+        BigDecimal paidAmount = claim.getCountyCourtJudgment().getPaidAmount();
+
+        if (!claim.getClaimData().getInterest().getType().equals(Interest.InterestType.NO_INTEREST)) {
+            BigDecimal interestRate = claim.getClaimData().getInterest().getRate();
+            BigDecimal interestAmount = interestCalculationService.calculateInterest(
+                claimAmount.subtract(paidAmount),
+                interestRate,
+                getInterestFromDate(claim),
+                toDate
+            );
+            return formatMoney(claimAmount.add(interestAmount).subtract(paidAmount));
+        }
+        return formatMoney(claimAmount.subtract(paidAmount));
+    }
+
+    private LocalDate getInterestFromDate(final Claim claim) {
+        if (claim.getClaimData().getInterest().getType().equals(InterestDate.InterestDateType.CUSTOM)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this call to "equals"; comparisons between unrelated types always return false. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS2159)